arch/arm/soc/st_stm32/stm32f7/dts.fixup

```diff
@@ -1,5 +1,53 @@
+#define CONFIG_NUM_IRQ_PRIO_BITS		ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define CONFIG_UART_STM32_USART_1_BASE_ADDRESS	ST_STM32_USART_40011000_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_1_BAUD_RATE	ST_STM32_USART_40011000_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_1_IRQ_PRI	ST_STM32_USART_40011000_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_1_NAME		ST_STM32_USART_40011000_LABEL
+#define USART_1_IRQ				ST_STM32_USART_40011000_IRQ_0
+
+#define CONFIG_UART_STM32_USART_2_BASE_ADDRESS	ST_STM32_USART_40004400_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_2_BAUD_RATE	ST_STM32_USART_40004400_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_2_IRQ_PRI	ST_STM32_USART_40004400_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_2_NAME		ST_STM32_USART_40004400_LABEL
+#define USART_2_IRQ				ST_STM32_USART_40004400_IRQ_0
+
+#define CONFIG_UART_STM32_USART_3_BASE_ADDRESS	ST_STM32_USART_40004800_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_3_BAUD_RATE	ST_STM32_USART_40004800_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_3_IRQ_PRI	ST_STM32_USART_40004800_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_3_NAME		ST_STM32_USART_40004800_LABEL
+#define USART_3_IRQ				ST_STM32_USART_40004800_IRQ_0
+
+#define CONFIG_UART_STM32_USART_4_BASE_ADDRESS	ST_STM32_USART_40004C00_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_4_BAUD_RATE	ST_STM32_USART_40004C00_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_4_IRQ_PRI	ST_STM32_USART_40004C00_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_4_NAME		ST_STM32_USART_40004C00_LABEL
+#define USART_4_IRQ				ST_STM32_USART_40004C00_IRQ_0
+
+#define CONFIG_UART_STM32_USART_5_BASE_ADDRESS	ST_STM32_USART_40005000_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_5_BAUD_RATE	ST_STM32_USART_40005000_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_5_IRQ_PRI	ST_STM32_USART_40005000_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_5_NAME		ST_STM32_USART_40005000_LABEL
+#define USART_5_IRQ				ST_STM32_USART_40005000_IRQ_0
+
+#define CONFIG_UART_STM32_USART_6_BASE_ADDRESS	ST_STM32_USART_40011400_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_6_BAUD_RATE	ST_STM32_USART_40011400_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_6_IRQ_PRI	ST_STM32_USART_40011400_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_6_NAME		ST_STM32_USART_40011400_LABEL
+#define USART_6_IRQ				ST_STM32_USART_40011400_IRQ_0
+
+#define CONFIG_UART_STM32_USART_7_BASE_ADDRESS	ST_STM32_USART_40007800_BASE_ADDRESS
+#define CONFIG_UART_STM32_USART_7_BAUD_RATE	ST_STM32_USART_40007800_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_7_IRQ_PRI	ST_STM32_USART_40007800_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_USART_7_NAME		ST_STM32_USART_40007800_LABEL
+#define USART_7_IRQ				ST_STM32_USART_40007800_IRQ_0
+
+#define CONFIG_UART_STM32_USART_8_BASE_ADDRESS	ST_STM32_USART_40007C00_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_8_BAUD_RATE	ST_STM32_USART_40007C00_CURRENT_SPEED
+#define CONFIG_UART_STM32_USART_8_IRQ_PRI	ST_STM32_USART_40007C00_IRQ_0_PRIORITY
```

Comment:
This should be CONFIG_UART_STM32_USART_8_BAUD_RATE instead of _PORT_.

Reply:
Thank you. I have fixed it.